crates/database/src/lib.rs

```diff
@@ -20,6 +23,11 @@ pub async fn clear_DANGER(ex: impl PgExecutor<'_> + Copy) -> sqlx::Result<()> {
     Ok(())
 }
 
+pub type Address = ByteArray<20>;
+pub type AppId = ByteArray<32>;
+pub type TransactionHash = ByteArray<32>;
+pub type OrderUid = ByteArray<56>;
```

Comment:
Not sure if plain aliases or newtypes are better here. Both give type safety and prevent copy paste errors but newtypes gives a little more typesafety while being a little harder to use. For now went with newtypes.

Reply:
Personally, I prefer type aliases here (unless you intend on adding methods to these).

My reasoning is that this is a representation of the Postgres storage that it is using.